### DISCUSSION TITLE
Please reopen #2612 as the fix was never commited to the code

### DISCUSSION THREAD

**Discussion by heldchen** [livewire/livewire#5550] [7. Bugs (archived)]
unfortunately the bug discussed in #2612 is still in the code as of today: 
https://github.com/livewire/livewire/blob/master/js/node_initializer.js#L78

the patch mentioned in the issue was never actually applied and as such the autofill is still broken.

```@@ -70,7 +70,7 @@
             || directive.modifiers.includes('lazy') ? 'change' : 'input'

         // If it's a text input and not .lazy, debounce, otherwise fire immediately.
-        let handler = debounceIf(hasDebounceModifier || (DOM.isTextInput(el) && !isLazy), e => {
+        let handler = debounceIf(hasDebounceModifier || (DOM.isTextInput(el) && (!isLazy || !el.wasRecentlyAutofilled)), e => {
             let model = directive.value
             let el = e.target```

**Comment by joshhanley** [comment]
We are now using discussions to track bugs and ideas, so won't be opening up the original issue, so we'll continue the discussion here.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

PRs are welcome to fix the bug if it was missed! 🙂
